### ISSUE TITLE
Shape input to `utils.resize` needs better documentation or input validation

### ISSUE THREAD

**Issue by msschwartz21** [vanvalenlab/deepcell-toolbox#14]
https://github.com/vanvalenlab/deepcell-toolbox/blob/7840c67706c0b79bfda4c50821e9475f72eacd31/deepcell_toolbox/utils.py#L293

In my current use case, I am inputing data with shape (1, 500, 500, 1) which needs to be reshaped to (1, 238, 238, 1). The `cv2.resize` function fails because it expects the shape to be (238, 238). We should either document this problem in the docstring or add `np.squeeze` to the shape input.

**Comment by msschwartz21** [comment]
@willgraf Do you have a preference? I think adding squeeze would be more robust.

**Comment by willgraf** [comment]
Yes, cv2 resize only works on 2D images (which is why we have the skimage version if the image is multi-channel). I think using squeeze is fine, but we must be careful about recasting back to the original shape in a general way. Will squeeze throw an error if there are no dims to squeeze out?
